### PR TITLE
fix(500-under-the-hood): Restructure "Using custom engine binaries"

### PR DESCRIPTION
**Pull request by janpio**
1. Fixes location of binaries
2. Restructures the instructions to set environment variables to avoid duplication of "check if it worked" step
3. Also fixes small details of the `-v` output

### CODE REVIEW

File: content/200-concepts/050-overview/500-under-the-hood.mdx [hunk 1]
```diff
@@ -26,7 +26,7 @@ As an example, Prisma Client connects to the [query engine](../components/prisma
 
 ### Using custom engine binaries
 
-By default, all binaries are automatically downloaded into the `node_modules/prisma` folder when you install or update `prisma`. You might want to use a [custom binary](https://github.com/prisma/prisma-engines) file if:
+By default, all binaries are automatically downloaded into the `node_modules/@prisma/engines` folder when you install or update `prisma`. You might want to use a [custom binary](https://github.com/prisma/prisma-engines) file if:
```
Comment (janpio) [inline content/200-concepts/050-overview/500-under-the-hood.mdx:29]: This is 1. from the PR description

File: content/200-concepts/050-overview/500-under-the-hood.mdx [hunk 2]
```diff
@@ -122,38 +88,41 @@ To set the an environment variable globally (in this example, `PRISMA_QUERY_ENGI
...
 
    ```sh highlight=2;normal
    Current platform     : darwin
    Query Engine         : query-engine d6ff7119649922b84e413b3b69660e2f49e2ddf3 (at /custom/my-query-engine-unix)
-   Migration Engine     : migration-engine-cli d6ff7119649922b84e413b3b69660e2f49e2ddf3 (at /myproject/node_modules/@prisma/engines/migration-engine-windows)
-   Introspection Engine : introspection-core d6ff7119649922b84e413b3b69660e2f49e2ddf3 (at /myproject/node_modules/@prisma/engines/introspection-engine-windows)
-   Format Binary        : prisma-fmt d6ff7119649922b84e413b3b69660e2f49e2ddf3 (at /myproject/node_modules/@prisma/engines/prisma-fmt-windows)
+   Migration Engine     : migration-engine-cli d6ff7119649922b84e413b3b69660e2f49e2ddf3 (at /myproject/node_modules/@prisma/engines/migration-engine-unix)
+   Introspection Engine : introspection-core d6ff7119649922b84e413b3b69660e2f49e2ddf3 (at /myproject/node_modules/@prisma/engines/introspection-engine-unix)
+   Format Binary        : prisma-fmt d6ff7119649922b84e413b3b69660e2f49e2ddf3 (at /myproject/node_modules/@prisma/engines/prisma-fmt-unix)
```
Comment (janpio) [inline content/200-concepts/050-overview/500-under-the-hood.mdx:112]: This is 3. from the PR description.

File: content/200-concepts/050-overview/500-under-the-hood.mdx [hunk 3]
```diff
@@ -122,38 +88,41 @@ To set the an environment variable globally (in this example, `PRISMA_QUERY_ENGI
...
-    Migration Engine     : migration-engine-cli 854c8ba7f0dce66f115af36af24e66989a8c02a1 (at node_modules\@prisma\engines\migration-engine-windows.exe)
-    Introspection Engine : introspection-core 854c8ba7f0dce66f115af36af24e66989a8c02a1 (at node_modules\@prisma\engines\introspection-engine-windows.exe)
-    Format Binary        : prisma-fmt 854c8ba7f0dce66f115af36af24e66989a8c02a1 (at node_modules\@prisma\engines\prisma-fmt-windows.exe)
+   Current platform     : windows
+   Query Engine         : query-engine d6ff7119649922b84e413b3b69660e2f49e2ddf3 (at c:\custom\my-query-engine-windows.exe)
+   Migration Engine     : migration-engine-cli d6ff7119649922b84e413b3b69660e2f49e2ddf3 (at c:\myproject\node_modules\@prisma\engines\migration-engine-windows.exe)
+   Introspection Engine : introspection-core d6ff7119649922b84e413b3b69660e2f49e2ddf3 (at c:\myproject\node_modules\@prisma\engines\introspection-engine-windows.exe)
+   Format Binary        : prisma-fmt d6ff7119649922b84e413b3b69660e2f49e2ddf3 (at c:\myproject\node_modules\@prisma\engines\prisma-fmt-windows.exe)
```
Comment (janpio) [inline content/200-concepts/050-overview/500-under-the-hood.mdx:123]: This is 3. from the PR description.

File: content/200-concepts/050-overview/500-under-the-hood.mdx [hunk 4]
```diff
@@ -38,73 +38,39 @@ Use the following environment variables to specify custom locations for your bin
 - `PRISMA_INTROSPECTION_ENGINE_BINARY` (Introspection engine)
 - `PRISMA_FMT_BINARY` (for `npx prisma format`)
 
-#### The `.env` file
+#### Setting the environment variable
 
-To set an environment variable in the [`.env` file](../concepts/components/prisma-schema#using-env-files):
+There are multiple ways to at an environment variables. Here are two ways:
```
Comment (mhwelander) [inline content/200-concepts/050-overview/500-under-the-hood.mdx]:
```suggestion
You can define environment variables globally on your machine or in the `.env` file.
```

File: content/200-concepts/050-overview/500-under-the-hood.mdx [hunk 5]
```diff
@@ -122,38 +88,41 @@ To set the an environment variable globally (in this example, `PRISMA_QUERY_ENGI
 
    </tab>
 
-   </TabbedContent>
+</TabbedContent>
+   
+#### Confirm the environment variable is picked up
```
Comment (mhwelander) [inline content/200-concepts/050-overview/500-under-the-hood.mdx]:
```suggestion
#### Test your environment variable
```

**Pull request merged**